src/main/java/uk/gov/hmcts/reform/iacasedocumentsapi/domain/service/DocumentReceiver.java

```diff
@@ -39,7 +39,7 @@ public DocumentWithMetadata receive(
             document,
             description,
             dateUploaded,
-            tag
+            tag,"test"
```

Comment:
it is better to leave empty "" for the suppliedBy field when it is not available to avoid misleading info.

Reply:
It is done. Thanks